strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/security/authentication/AuthenticatorsConfigController.java

```diff
@@ -27,49 +35,41 @@
         extends BaseController
 {
 
-    static final String SUCCESSFUL_REORDER = "Re-order succeeded.";
-
-    static final String FAILED_REORDER = "Could not reorder authenticators registry.";
-
-    static final String SUCCESSFUL_RELOAD = "Reload succeeded.";
+    public static final String SUCCESSFUL_UPDATE = "Update succeeded.";
+    
+    public static final String SUCCESSFUL_REORDER = "Re-order succeeded.";
 
-    static final String FAILED_RELOAD = "Could not reload authenticators registry.";
+    public static final String FAILED_REORDER = "Could not reorder authentication items.";
+    
+    public static final String FAILED_UPDATE= "Could not update authentication configuration.";
 
-    private final AuthenticatorsRegistry authenticatorsRegistry;
+    public static final String SUCCESSFUL_RELOAD = "Reload succeeded.";
```

Comment:
Cosmetic: `Re-load successful.` (Although I think it should be more specific -- what was re-loaded? When you're looking at log files it would be hard to tell).

Reply:
done